build_projects/dotnet-cli-build/DotNetRestore.cs

```diff
@@ -12,9 +12,11 @@ protected override string Command
 
         protected override string Args
         {
-            get { return $"{GetProjectPath()} {GetSource()} {GetPackages()} {GetSkipInvalidConfigurations()} {GetRuntime()}"; }
+            get { return $"{GetProjectPath()} {GetSource()} {GetPackages()} {GetSkipInvalidConfigurations()} {GetRuntime()} {GetAdditionalParameters()}"; }
```

Comment:
yea, we return null from these helpers if they don't have a value across all these tasks, so maintaining consistency.